src/main/scala/scala/collection/next/package.scala

```diff
@@ -19,4 +19,14 @@ package object next {
     col: IterableOnceOps[A, CC, C]
   ): NextIterableOnceOpsExtensions[A, CC, C] =
     new NextIterableOnceOpsExtensions(col)
+
+  implicit final class OptionOpsExtensions[A](val v: Option[A]) extends AnyVal {
```

Comment:
`Option` is in package `scala`, not `scala.collection`, so I'd actually put this extension method in `scala.next` rather than `scala.collection.next`

Reply:
I've put the extension in `scala.next` and added the proposed tweak so the compiler can't coerce a specific implicit to be used.

Reply:
thanks! watch someone who actually knows how the compiler works tell me I was wrong now >.>